splunk-quarkus/pom.xml

```diff
@@ -135,7 +135,6 @@
         <dependency>
             <groupId>io.micrometer</groupId>
             <artifactId>micrometer-registry-prometheus</artifactId>
-            <version>1.8.3</version>
         </dependency>
```

Comment:
Did you try replacing this dependency with this?
```
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-micrometer-registry-prometheus</artifactId>
</dependency>
```

Reply:
No I haven't. Is this the correct one?